lib/docker.js

```diff
@@ -527,6 +527,9 @@ function main(options, configuration) {
     .then(function() {
         if (opts.deploy) {
             return updateDeploy();
+        } else if (opts.tests) {
+            // append a volume so that tests could create files in container
+            return startContainer(['-v', process.cwd() + ':/opt/service']);
```

Comment:
When running tests, we already use `/opt/service` as the root dir for the service, where the node modules are reinstalled from scratch. Simply attaching the `cwd` over it means that incompatibilities may arise if the host and guest OSes are different, as binary modules might not be compatible. A better solution might be to `chown` and `chmod` that directory so that tests may create files as needed.